entrypoint/scripts/minio.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+myd=$(readlink -f $(dirname $0))
+tmpd=$(mktemp -d --tmpdir=/tmp)
+finalp=$(cd ${myd}/.. && mkdir -p bin && readlink -f bin)
+if [ -f "${finalp}/minio" ]; then
+    exit 0
+fi
+
+pushd ${tmpd}
+
+trap "rm -rf ${tmpd}" EXIT
+
+git clone -q --recursive --depth 1 --branch RELEASE.2020-10-18T21-54-12Z https://github.com/minio/minio
```

Comment:
This makes the git repo less "self contained" - what about using a git submodule?  Or packaging I guess.

Reply:
Or, use the upstream container as part of a pod (i.e. we have a cosa container and mini container in one pod)?

Reply:
I _really_ wanted to use the upstream but Multiarch.

Reply:
and yeah, using submodules is definately the better, cleaner way.

Reply:
Re multiarch yeah, but that's something we need to deal with in general.